README.md

```diff
@@ -1,11 +1,13 @@
 # rubocop-diff
 
-This github action runs rubocop under 2 differents commits and tells you if you introduced new offences or reduced offences
+This github action runs rubocop under 2 differents commits and tells you if you have introduced new offenses or reduced offenses
 
```

Comment:
Switch to [US spelling](https://english.stackexchange.com/questions/16668/) is intentional?  Nothing wrong with that in my view (I guess most potential users will be yanks).